src/axom/slic/core/Logger.hpp

```diff
@@ -166,6 +166,19 @@ class Logger
    */
   LogStream* getStream(message::Level level, int i);
 
+  ///@{
+  //! \name Collective Methods
+  //!
+  //! \attention These methods are collective operations.
+  //! All ranks in the user-supplied communicator must call the method
+  //! when used within an MPI distributed environment.
+  //! Additionally, for logMessage:
```

Comment:
I like this grouping!  Thank you.  Rephrasing suggestion:
```suggestion
  //! The logMessage method is collective if:
```